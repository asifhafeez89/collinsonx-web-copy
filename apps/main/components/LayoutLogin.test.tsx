import * as React from 'react';
import renderer from 'react-test-renderer';
import SignupUser from '../pages/signup-user';

describe('<SignupUser />', () => {
  it('renders outline variant', () => {
    const tree = renderer.create(<SignupUser />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
