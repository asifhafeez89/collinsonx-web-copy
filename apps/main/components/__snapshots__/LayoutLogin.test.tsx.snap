// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutLogin /> renders outline variant 1`] = `
<div
  className="mantine-Container-root mantine-1ne5wbx"
>
  <div
    className="mantine-Stack-root mantine-f7tv2s"
  >
    <div
      className="mantine-8od8ev"
    >
      svg
       
    </div>
    <div />
  </div>
</div>
`;
